docs/guide/service/nlb.md

```diff
@@ -87,20 +162,40 @@ service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 ## Subnet tagging requirements
 See [Subnet Discovery](../../deploy/subnet_discovery.md) for details on configuring ELB for public or private placement.
 
-
 ## Security group
-NLB does not currently support managed security groups. For ingress access, the controller adds inbound rules to the node security group for the instance mode, or the ENI security group for the IP mode. In case of multiple
-security groups, the controller expects only one security group tagged with the cluster name as follows:
+AWS currently does not support attach security groups to NLB. To allow inbound traffic from NLB, the controller automatically adds inbound rules to worker node security groups by default.
```

Comment:
`adds inbound rules to the worker node security groups by default`

Reply:
+1